packages/components/src/RadarChart/__snapshots__/PieChart.test.js.snap

```diff
@@ -0,0 +1,50 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why do we have those `PieChart` snapshots ?

Reply:
Doesn't seem very useful does it! Work in progress, they'll be gone soon.

Reply:
Piechart tests removed and replaced with new radar chart tests.